test.js

```diff
@@ -2,6 +2,17 @@ import { test } from 'tapzero'
 
 import { jams, read } from './jams.js'
 
+import {readFileSync} from 'fs'
+
+test('json comparison', t=> {
+    const json_o = JSON.parse(readFileSync("./exampledata/example.json"))
+    const jams_o = jams(String(readFileSync('./exampledata/example.jams')))
```

Comment:
How come we need to have this `String` cast? Probably we should do this inside of `jams`, the usage should be just like JSON.parse

Reply:
Agreed. Will move.

Reply:
addressed

Reply:
It was needed because easygram wasn't parsing the Buffer correctly.